website/api/options-backButton.mdx

```diff
@@ -9,42 +9,55 @@ Controls the back button styling.
 ```js
 const options = {
   topBar: {
-    backButton: {}
-  }
-}
+    backButton: {},
+  },
+};
 ```
 
 ### `color`
+
 Change the back button color. This will change the text color as well.
 
 | Type  | Required | Platform |
 | ----- | -------- | -------- |
 | color | No       | Both     |
 
 ### `icon`
+
 Change the default back button icon.
 
 | Type   | Required | Platform |
 | ------ | -------- | -------- |
 | number | No       | Both     |
 
 ### `showTitle`
+
 Show or hide the text displayed next to the back button.
 
 | Type   | Required | Platform |
 | ------ | -------- | -------- |
 | number | No       | iOS      |
 
 ### `title`
+
 Change the text displayed next to the title. Usually the back button shows the title of the previous screen.
 
 | Type   | Required | Platform |
 | ------ | -------- | -------- |
 | string | No       | iOS      |
 
 ### `visible`
+
 Hide or show the back button.
 
 | Type    | Required | Platform |
 | ------- | -------- | -------- |
 | boolean | No       | Both     |
+
+### `testID`
+
+Used to interact with the back button in e2e tests.
+
+| Type    | Required | Platform |
+| ------- | -------- | -------- |
+| boolean | No       | Both     |
```

Comment:
Type is string.
Also, why are there so many formatting changes in the file? Is it because of Prettier?

Reply:
Yes